packages/runtime-common/realm-index-updater.ts

```diff
@@ -145,7 +145,7 @@ export class RealmIndexUpdater {
       let job = await this.#queue.publish<IncrementalResult>({
         jobType: `incremental-index`,
         concurrencyGroup: `indexing:${this.#realm.url}`,
-        timeout: 4 * 60,
+        timeout: 60,
```

Comment:
generally incremental indexing should take less than a second, so a 1 minute timeout seems more than enough.